patches/minecraft/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/server/network/ServerGamePacketListenerImpl.java
 +++ b/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+@@ -171,6 +_,7 @@
+ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, ServerGamePacketListener {
+    static final Logger f_9744_ = LogUtils.getLogger();
+    private static final int f_143608_ = 15000;
++   @Deprecated // Forge: See Reach Distance / Attack Range attributes.
```

Comment:
This should be a javadoc comment with (fully-qualified) references to the attributes in `ForgeMod`.